RELEASE.md

```diff
@@ -0,0 +1,12 @@
+# Release
+
+Releasing is controlled by leveraging git tags and releases.
+
+We follow a semantic versioning schema prefixing with a `v`. e.g. `v1.2.3`.
```

Comment:
Or we could drop the `v`? It's more consistent with what we currently have without. What's your opinion?